### PR TITLE
Prison Details page & Enhancements

### PR DESCRIPTION
**Pull request by AsalaKM**
view prison details page
allow  edit the details

### CODE REVIEW

File: src/client/src/components/pages/dashboard/index.jsx [hunk 1]
```diff
@@ -63,8 +63,8 @@ class index extends Component {
         course: countCourse,
         worker: countWorker,
         employment: [
-          { decription: "Percentage of participants who have reoffinding state and employed", percentage: ReoffengingEmployedAvg },
-          { decription: "Percentage of participants who have reoffinding state and unemployed", percentage: ReoffengingUnemployedAvg }
+          { decription: "Percentage of participants who have reoffending state and employed", percentage: ReoffengingEmployedAvg },
```
Comment (maxgerber) [inline src/client/src/components/pages/dashboard/index.jsx]: :tada: :tada: :tada:

File: src/client/src/components/pages/participants/participantDetails/staticData.js [hunk 2]
```diff
@@ -41,7 +42,14 @@ export const fields = [
...
       staticField: true
-    },  {
+    },  
+    {
+      tag: "Button",
+      value: "Prison Details",
+      display: "none",
+      color: "#FF4800",
+      staticField: true
+    },{
```
Comment (maxgerber) [inline src/client/src/components/pages/participants/participantDetails/staticData.js]: ``` },{``` looks wrong and inconsistent with the other objects in this array. This suggests to me that ESLint is not working, as the formatting stuff should be fixed automatically

File: src/client/src/components/pages/participants/prisonerDetails/index.jsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
+import React, { Component } from "react";
+import {
+  state as initialState,
+  fields as fieldSet,
+} from "./staticData";
+import Form from "./../../../abstract/Form";
+import Footer from '../../../abstract/footer';
```
Comment (maxgerber) [inline src/client/src/components/pages/participants/prisonerDetails/index.jsx]: Single or double-quotes for strings? You can set this in ESLint.

File: src/client/src/components/pages/participants/prisonerDetails/index.jsx [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+import axios from "axios";
+import swal from "sweetalert2";
+import contextHoc from './../../../abstract/HOC/contextHoc';
+
+class PresionDetails extends Component {
```
Comment (maxgerber) [inline src/client/src/components/pages/participants/prisonerDetails/index.jsx]: This should be PrisonDetails!

```suggestion
class PrisonDetails extends Component {
```

File: src/client/src/components/pages/participants/prisonerDetails/index.jsx [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+          onChange={this.onChange}
+          btnEvents={[this.onSubmit, this.goBack]}
+        />
+        <Footer />
+      </div>
+    );
+  }
+}
+
+export default contextHoc(PresionDetails);
```
Comment (maxgerber) [inline src/client/src/components/pages/participants/prisonerDetails/index.jsx]:
```suggestion
export default contextHoc(PrisonDetails);
```

File: src/client/src/components/pages/participants/participantDetails/staticData.js (hunk 2 above)
Comment (AsalaKM) [reply src/client/src/components/pages/participants/participantDetails/staticData.js]: i do it

File: src/client/src/components/pages/participants/prisonerDetails/index.jsx (hunk 4 above)
Comment (AsalaKM) [reply src/client/src/components/pages/participants/prisonerDetails/index.jsx]: yeah you are right

File: src/client/src/components/pages/participants/prisonerDetails/index.jsx (hunk 5 above)
Comment (AsalaKM) [reply src/client/src/components/pages/participants/prisonerDetails/index.jsx]: :+1:

**Pull request merged**